WorkloadTools/Listener/ExtendedEvents/StreamXEventDataReader.cs

```diff
@@ -59,6 +70,23 @@ public override void ReadEvents()
                             commandText = (string)TryGetValue(evt, FieldType.Field, "batch_text");
                             workloadEvent.Type = WorkloadEvent.EventType.BatchCompleted;
                         }
+                        else if (evt.Name == "login")
+                        {
+                            bool vIsCached = Convert.ToBoolean(TryGetValue(evt, FieldType.Field, "is_cached"));
+                            if (!vIsCached) /* If is not cached then consider it a new login */
+                            {
+                                workloadEvent.Type = WorkloadEvent.EventType.RPCStarting;
```

Comment:
Why reusing RPCStarting for the login event? I would have used a specialized event type in the Enum